protocol/v2/ssv/validator/committee.go

```diff
@@ -36,7 +37,8 @@ type Committee struct {
 	//sharesMtx sync.RWMutex
 	Shares map[phase0.ValidatorIndex]*spectypes.Share
 
-	Operator                *spectypes.Operator
```

Comment:
In spec, we also renamed the attribute `Operator` to `CommitteeMember`. But this was last week so perhaps this change is not in the version that was used as reference for this PR